groovy/inductor_dashboard.groovy

```diff
@@ -0,0 +1,376 @@
+                [$class: 'StringParameterValue', name: 'NODE_LABEL', value: 'Docker'],
+                [$class: 'StringParameterValue', name: 'BASE_IMAGE', value: "${BASE_IMAGE}"],                
+                [$class: 'StringParameterValue', name: 'PT_REPO', value: "${PT_REPO}"],
+                [$class: 'StringParameterValue', name: 'PT_BRANCH', value: "${PT_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'PT_COMMIT', value: "${PT_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_VISION_BRANCH', value: "${TORCH_VISION_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_VISION_COMMIT', value: "${TORCH_VISION_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_TEXT_BRANCH', value: "${TORCH_TEXT_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_TEXT_COMMIT', value: "${TORCH_TEXT_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_BRANCH', value: "${TORCH_DATA_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_COMMIT', value: "${TORCH_DATA_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_BRANCH', value: "${TORCH_AUDIO_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_COMMIT', value: "${TORCH_AUDIO_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_BRANCH', value: "${TORCH_BENCH_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_COMMIT', value: "${TORCH_BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'BENCH_COMMIT', value: "${BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'tag', value: "${image_tag}"],
+            ]
+            task_status = image_build_job.result
+            task_number = image_build_job.number
+            withEnv(["task_status=${task_status}","task_number=${task_number}"]) {
+                sh '''#!/bin/bash
+                    tag=${image_tag}
+                    old_container=`docker ps |grep pt_inductor |awk '{print $1}'`
+                    if [ -n "${old_container}" ]; then
+                        docker stop $old_container
+                        docker rm $old_container
+                        docker container prune -f
+                    fi
+                    old_image_id=`docker images|grep pt_inductor|grep ${tag}|awk '{print $3}'`
+                    old_image=`echo $old_image_id| awk '{print $1}'`
+                    if [ -n "${old_image}" ]; then
+                        docker rmi -f $old_image
+                    fi
+                    if [ ${task_status} == "SUCCESS" ]; then
+                        docker system prune -f
+                        docker login ccr-registry.caas.intel.com
+                        docker pull ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}

```

Comment:
We can define the `ccr-registry.caas.intel.com/pytorch/pt_inductor` as `DOCKER_IMAGE_NAMESPACE ` for easy modify in the future

Reply:
OK